src/plugins/blocksmith/utils/getLines.js

```diff
@@ -0,0 +1,57 @@
+import DraftOffsetKey from 'draft-js/lib/DraftOffsetKey';
+
+/**
+ * Calculates and returns an array of the individual lines of a text block. Useful for
+ * some keyboard navigation behavior.
+ *
+ * @param {EditorState} editorState - a state instance of a DraftJs Editor
+ *
+ * @returns {boolean}
```

Comment:
@efjacobson - please change the return type to `Array`

Reply:
@arunkarnati done